src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -53,6 +53,25 @@ const timezoneUISchema = {
 
 const shouldRenderAsRadio = (name) => name.indexOf('methodType') >= 0 || name.indexOf('channel') >= 0;
 
+const populateUISchemaForDisplay = (uiSchema, ionField) => {
+  let display = ionField.value.value;
```

Comment:
nit, looks like can use `const display` instead of `let display`

Reply:
should be `ionField?.value?.value;`

Reply:
I have replaced the line of code by `const display = ionField?.value?.value;`